subsys/bluetooth/controller/ll_sw/nordic/lll/lll_sync_iso.c

```diff
@@ -1173,7 +1416,21 @@ static void isr_rx(void *param)
 	/* Calculate ahead the next subevent channel index */
 	const uint16_t event_counter = (lll->payload_count / lll->bn) - 1U;
 
-	next_chan_calc(lll, event_counter, data_chan_id);
+	if (false) {
+
+#if defined(CONFIG_BT_CTLR_SYNC_ISO_SEQUENTIAL)
+	} else if (lll->bis_spacing >= (lll->sub_interval * lll->nse)) {
+		next_chan_calc_seq(lll, event_counter, data_chan_id);
+#endif /* CONFIG_BT_CTLR_SYNC_ISO_SEQUENTIAL */
+
+#if defined(CONFIG_BT_CTLR_SYNC_ISO_INTERLEAVED)
+	} else if (lll->bis_spacing < (lll->sub_interval * lll->nse)) {
```

Comment:
Ditto

This structure seem to exist multiple places. Couldn't this be put into a function to avoid code duplication?